### ISSUE TITLE
Change autocomplete template for bool operators

### ISSUE THREAD

**Issue by spaduret** [AntonShuvaev/elasticsearch4idea#124]
**Update autocomplete template for bool operators**

This is not really a bug but when you build a bool query with must/should... operators autocomplete creates is with a `{}` brackets:
```
"query": {
  "bool": {
    "must": {
      "exists": {
        "field": "id"
      }
    }
  }
}
```

It will be nice to change it to follow Kibana behavior and use `[]` brackets because most of the time multiple conditions are used.

**Environment information:**
- OS: Windows 10
- IDE version: DataGrip 2023.2
- plugin version: 2023.2.5-232

**Comment by AntonShuvaev** [comment]
Thanks for the suggestion. I will do it in the next version.

**Comment by AntonShuvaev** [comment]
Fixed in 2023.2.6

**Comment by spaduret** [comment]
Thank you very much!
